recipes/custom-inherit/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "custom-inherit" %}
+{% set version = "2.2.2" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/c/{{ name }}/custom_inherit-{{ version }}.tar.gz"
+  sha256: 83c048bc3415a9e38e44e78dbe231f837aa3d4fd91b4e71443b6f6e38034f583
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - custom_inherit
+
+about:
+  home: "https://github.com/rsokl/custom_inherit"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.md"
+  summary: "A Python package that provides customized docstring inheritance schemes between derived classes and their parents."
+  doc_url: "https://github.com/rsokl/custom_inherit/blob/master/README.md"
+  dev_url: ""
```

Comment:
I would remove this, rather than having an empty string

Reply:
```suggestion

```

Reply:
this looks good too